packages/ethereum/hardhat.config.ts

```diff
@@ -43,37 +44,42 @@ export function expandVars(input: string, vars: { [key: string]: any }) {
   })
 }
 
-// Copied from `core` to prevent from ESM import issues
-// ESM support requires code changes within `hardhat-core`
-type NetworkOptions = {
-  id: string
-  description: string
-  chain_id: number // >= 0
-  live: boolean
-  hardhat_deploy_gas_price: string // Gas price as either a number string '11' or a value which should be converted like '1 gwei'. Used in hardhat-deploy plugin.
-  default_provider: string // a valid HTTP url pointing at a RPC endpoint
-  etherscan_api_url?: string // a valid HTTP url pointing at a RPC endpoint
-  max_fee_per_gas: string // The absolute maximum you are willing to pay per unit of gas to get your transaction included in a block, e.g. '10 gwei'
-  max_priority_fee_per_gas: string // Tips paid directly to miners, e.g. '2 gwei'
-  native_token_name: string
-  hopr_token_name: string
-  tags: string[]
-}
 // rest
 import { task, types, extendEnvironment, subtask } from 'hardhat/config'
 import { writeFileSync, realpathSync } from 'fs'
 import { TASK_TEST_SETUP_TEST_ENVIRONMENT } from 'hardhat/builtin-tasks/task-names'
 import { HARDHAT_NETWORK_NAME } from 'hardhat/plugins'
-import { TASK_DEPLOY_RUN_DEPLOY } from 'hardhat-deploy'
 import stake, { StakeOpts } from './tasks/stake'
 import { MIN_STAKE } from './utils/constants'
+import type { BigNumber } from 'ethers'
 
 const { DEPLOYER_WALLET_PRIVATE_KEY, ETHERSCAN_KEY, HOPR_ENVIRONMENT_ID, HOPR_HARDHAT_TAG } = process.env
 
 const PROTOCOL_CONFIG = require('../core/protocol-config.json')
 
+function ethereumUnitToNumberString(fee: string, hre: HardhatRuntimeEnvironment): BigNumber {
+  const parsedGasPrice = fee.split(' ')
```

Comment:
better to not use `gas price` in the variable as this seems to be a generic function to deal with number-strings